retriever/engines/postgres.py

```diff
@@ -1,7 +1,7 @@
 import os
 import sys
 import subprocess
-
+from osgeo import gdal
```

Comment:
@dassaniansh, so your tests are failing because `osgeo` is not installed, if you haven't written any tests for the new changes, put this import in a try-except block to avoid the failing tests for the time being.

Or you could add the `osgeo` module to the `python-package.yml`

Reply:
For now we can add a try block, but later we need to install gdal and make sure we set the paths for both `python-package.yml` (Linux) and appveyor.yml (Windows)